osgar/tools/lidarview.py

```diff
@@ -418,8 +421,8 @@ def lidarview(gen, caption_filename, callback=False, out_video=None, jump=None):
 
         while True:
             caption = caption_filename + ": %s" % timestamp
-            if title is not None:
-                caption += ' (' + str(title) + ')'
+            for t in title:
+                caption += ' (' + str(t) + ')'
```

Comment:
we could reduce the amount of brackets for the list, but ... that is just a detail, now separation is clear